src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java

```diff
@@ -27,17 +25,18 @@ public void testComparison() throws IOException {
 
         expected = "<HTML><HEAD><SCRIPT language=\"JavaScript\" src=\"a.js\" ></SCRIPT></HEAD>"
                    + "<BODY id=\"5\"><P>abc</P><DIV id=\"frameBottom\"><DIV></DIV></DIV></BODY></HTML>";
-        actual = expected.replace("<HEAD>", "    <HEAD>    \t" + EOL);
+        actual = expected.replace("<HEAD>", "    <HEAD>    \t" + System.lineSeparator());
         HtmlHelper.assertSameHtml(expected, actual, false);
 
         //change attribute order
         actual = expected.replace("language=\"JavaScript\" src=\"a.js\"", "  src=\"a.js\"   language=\"JavaScript\"  ");
         HtmlHelper.assertSameHtml(expected, actual, false);
 
-        actual = expected.replace("<P>", "<P>\n\n" + EOL + EOL);
+        actual = expected.replace("<P>", "<P>\n\n" + System.lineSeparator() + System.lineSeparator());
         HtmlHelper.assertSameHtml(expected, actual, false);
 
-        actual = expected.replace("<DIV></DIV></DIV>", EOL + EOL + "\n<DIV>\n\n</DIV></DIV>\n\n" + EOL);
+        actual = expected.replace("<DIV></DIV></DIV>", System.lineSeparator()
```

Comment:
Break before the first `System.lineSeparator()`.

Reply:
Actually I tried this but line size is increasing and going more than 125 chars per line

Reply:
That's not true?

Reply:
Sorry, my bad